sphinx_rtd_theme/footer.html

```diff
@@ -32,5 +32,9 @@
   {% trans %}Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>{% endtrans %}.
   {%- endif %}
 
+  {%- if commit %}
+  Build commit: <code>{{ commit }}</code>
```

Comment:
So a couple issues with this seemingly small change:

 * Text placement at the end of the footer line looks awkward, I think "Build XXXX, built with Sphinx" or something more graceful would work better.
 * Is this the SHA short hash? We don't want the long hash here
 * Do we want to take the chance here to link back to the build output, as I mentioned in rtfd/readthedocs.org#1357?
 * Do we just make this the build id instead, and link back to the build page?

OR...

Put in this information in the flyout instead?